coordinator/persistence/aggregation/src/integrationTest/kotlin/net/consensys/zkevm/persistence/dao/aggregation/AggregationsPostgresDaoTest.kt

```diff
@@ -293,6 +293,90 @@ class AggregationsPostgresDaoTest : CleanDbTestSuiteParallel() {
     }
   }
 
+  @Test
+  fun findConsecutiveProvenBlobsReturnsFullListOfProvenBlobsWithProvenBatches() {
+    val batch1 = createBatch(0L, 10L)
+    val batch2 = createBatch(11L, 20L)
+    val batch3 = createBatch(21L, 30L)
+
+    val blob1 = createBlobRecordFromBatches(listOf(batch1))
+    val blob2 = createBlobRecordFromBatches(listOf(batch2, batch3))
+
+    val batches = listOf(batch1, batch2, batch3)
+    val blobs = listOf(blob1, blob2)
+
+    batches.forEach { insertBatch(it).get() }
+    blobs.forEach { insertBlob(it).get() }
```

Comment:
this is slow. can't they be inserted in parallel ino the Db with `SafeFuture.collectAll`? 

I keep on calling on this :(